src/Mindee/Product/Fr/BankAccountDetails/BankAccountDetailsV2.cs

```diff
@@ -0,0 +1,13 @@
+using Mindee.Http;
+using Mindee.Parsing.Common;
+
+namespace Mindee.Product.Fr.BankAccountDetails
+{
+    /// <summary>
+    /// The definition for bank_account_details v2.
+    /// </summary>
+    [Endpoint("bank_account_details", "2")]
+    public class BankAccountDetailsV2 : Inference<BankAccountDetailsV2Document, BankAccountDetailsV2Document>
```

Comment:
Could we make all product classes `sealed`? Aside from the mostly negligible performance increase, it'd make it more clear that products don't inherit from each other, even between versions (e.g. BankAccountDetailsV2 does NOT inherit from BankAccountDetailsV1, etc.).

Reply:
will do in other PR